src/navigation/AppStackNavigator.js

```diff
@@ -58,6 +60,7 @@ export const AppStackNavigator = (headerRight = true) => {
         options={screenOptionsWithSettings(headerRight)}
       />
       <Stack.Screen name="BookmarkCategory" component={BookmarkCategoryScreen} />
+      <Stack.Screen name="Category" component={IndexScreen} />
```

Comment:
shouldn't be included here?

Reply:
yes it should. i moved it up to sort by `name`

Reply:
ah i thought it is added here out of context. my fault.